src/quo2/components/messages/system_message/view.cljs

```diff
@@ -177,13 +150,38 @@
 
 (def system-message-pinned (quo.theme/with-theme system-message-pinned-internal))
 
+(defn f-system-message
+  [{:keys [type animate-bg-color? on-long-press] :as data}]
+  (let [animated-bg-color (reanimated/use-shared-value
+                           style/system-message-deleted-animation-start-bg-color)
+        wrapper           (if (or on-long-press animate-bg-color?) reanimated/touchable-opacity rn/view)
+        animated-style    (reanimated/apply-animations-to-style
+                           {:background-color animated-bg-color}
+                           (assoc style/system-message-wrapper
+                                  :background-color
+                                  animated-bg-color))]
+
+    (when animate-bg-color?
+      (reanimated/animate-shared-value-with-delay
+       animated-bg-color
+       style/system-message-deleted-animation-end-bg-color
+       0
+       :linear
+       1000))
+
+    [wrapper
+     {:style         (if animate-bg-color?
+                       animated-style
+                       style/system-message-wrapper)
+      :on-long-press on-long-press}
```

Comment:
are you sure on-long-press and background are needed for all system messages?

Reply:
where does this animated background come from? do we have documentation for this?

Reply:
They are not needed by other system messages, only deleted (for me) message
it's just the UI already done in system message, long press and animation both affect the UI so I choose to put them in system message

animated background design is at https://www.figma.com/file/PPWkgOYlZZDxZv5SDGsZVV/Posts-%26-Attachments-for-Mobile?type=design&node-id=5713-624109&mode=design&t=TVdUm1nRzpSVoy6k-4